lib/cadet/chat/room.ex

```diff
@@ -0,0 +1,100 @@
+defmodule Cadet.Chat.Room do
+  @moduledoc """
+  Contains logic pertaining to chatroom creation to supplement ChatKit, an external service engaged for Source Academy.
+  ChatKit's API can be found here: https://pusher.com/docs/chatkit
+  """
+
+  require Logger
+
+  import Ecto.Query
+
+  alias Cadet.Repo
+  alias Cadet.Assessments.{Answer, Submission}
+  alias Cadet.Accounts.User
+  alias Cadet.Chat.Token
+
+  @instance_id :cadet |> Application.fetch_env!(:chat) |> Keyword.get(:instance_id)
+
+  @doc """
+  Creates a chatroom for every answer, and updates db with the chatroom id.
+  Takes in Submission struct
+  """
+  def create_rooms(
+        submission = %Submission{
+          id: submission_id,
+          student_id: student_id
+        }
+      ) do
+    user = User |> where(id: ^student_id) |> Repo.one()
+
+    Answer
+    |> where(submission_id: ^submission_id)
+    |> Repo.all()
+    |> Enum.filter(fn answer ->
+      answer.comment == "" or answer.comment == nil
```

Comment:
Can we remove the `answer.comment == ""` check, it doesn't seem to do anything. Let's keep it simple!

Reply:
What are the cases where `answer.comment == ""`?

Reply:
I wasn't sure what `comment` was initialised with so I checked for empty strings. Removed now!